[2024-12-10 18:13:19,545] INFO [logger.logger.kafka_producer:40] Kafka Producer: data : {"data": "zwasdfsdffsdffd"}
[2024-12-10 18:13:19,681] INFO [logger.logger.kafka_producer:40] Kafka Producer: data : {"data": "zwasdfsdffsdffd"}
[2024-12-10 18:13:38,534] INFO [logger.logger.kafka_consumer:23] Kafka Consumer: Started
[2024-12-10 18:15:25,241] INFO [logger.logger.kafka_consumer:26] Kafka Consumer: Key: None - Value: b'{"data": "zwasdfsdffsdffd"}'
[2024-12-10 18:15:26,377] INFO [logger.logger.kafka_consumer:26] Kafka Consumer: Key: None - Value: b'{"data": "zwasdfsdffsdffd"}'
[2024-12-10 18:15:27,015] ERROR [aiokafka.consumer.group_coordinator._do_commit_offsets:1123] OffsetCommit failed for group notif_group due to group error ([Error 25] UnknownMemberIdError: notif_group), will rejoin
[2024-12-10 18:15:27,015] WARNING [aiokafka.consumer.group_coordinator._maybe_do_autocommit:970] Auto offset commit failed: [Error 25] UnknownMemberIdError: notif_group
[2024-12-10 18:15:27,017] WARNING [aiokafka.consumer.group_coordinator._do_heartbeat:860] Heartbeat failed: local member_id was not recognized; resetting and re-joining group
[2024-12-10 18:15:27,017] ERROR [aiokafka.consumer.group_coordinator._heartbeat_routine:795] Heartbeat session expired - marking coordinator dead
[2024-12-10 18:15:27,017] WARNING [aiokafka.consumer.group_coordinator.coordinator_dead:531] Marking the coordinator dead (node 1)for group notif_group.
[2024-12-10 18:36:38,383] INFO [logger.logger.kafka_consumer:22] Kafka Consumer: Started
[2024-12-10 18:37:14,472] ERROR [asyncio.default_exception_handler:1758] Task was destroyed but it is pending!
task: <Task pending name='Task-3' coro=<kafka_consumer() running at /home/mind/Source/rabbit-mq/src/api/v1/services/kafka/kafka.py:30> wait_for=<Future pending cb=[Task.task_wakeup()]> cb=[gather.<locals>._done_callback() at /usr/lib/python3.10/asyncio/tasks.py:720]>
[2024-12-10 18:37:18,607] INFO [logger.logger.kafka_consumer:22] Kafka Consumer: Started
[2024-12-10 18:37:44,106] ERROR [asyncio.default_exception_handler:1758] Task was destroyed but it is pending!
task: <Task pending name='Task-3' coro=<kafka_consumer() running at /home/mind/Source/rabbit-mq/src/api/v1/services/kafka/kafka.py:30> wait_for=<Future pending cb=[Task.task_wakeup()]> cb=[gather.<locals>._done_callback() at /usr/lib/python3.10/asyncio/tasks.py:720]>
[2024-12-10 18:38:03,590] INFO [logger.logger.kafka_consumer:22] Kafka Consumer: Started
[2024-12-10 18:41:19,659] INFO [logger.logger.kafka_consumer:22] Kafka Consumer: Started
[2024-12-10 18:41:36,668] INFO [logger.logger.kafka_consumer:22] Kafka Consumer: Started
[2024-12-10 18:41:50,678] INFO [logger.logger.kafka_consumer:22] Kafka Consumer: Started
[2024-12-10 18:42:25,712] INFO [logger.logger.kafka_consumer:22] Kafka Consumer: Started
[2024-12-10 18:47:24,081] INFO [logger.logger.kafka_consumer:22] Kafka Consumer: Started
[2024-12-10 18:54:01,422] INFO [logger.logger.kafka_consumer:22] Kafka Consumer: Started
[2024-12-10 18:56:46,737] INFO [logger.logger.kafka_consumer:22] Kafka Consumer: Started
[2024-12-10 18:57:30,777] INFO [logger.logger.kafka_consumer:22] Kafka Consumer: Started
[2024-12-10 19:00:10,843] INFO [logger.logger.kafka_consumer:22] Kafka Consumer: Started
[2024-12-10 19:00:48,892] INFO [logger.logger.kafka_consumer:22] Kafka Consumer: Started
[2024-12-10 19:01:23,928] INFO [logger.logger.kafka_consumer:22] Kafka Consumer: Started
